api/api_full.go

```diff
@@ -163,6 +166,8 @@ type FullNode interface {
 	PaychVoucherAdd(context.Context, address.Address, *paych.SignedVoucher, []byte, types.BigInt) (types.BigInt, error)
 	PaychVoucherList(context.Context, address.Address) ([]*paych.SignedVoucher, error)
 	PaychVoucherSubmit(context.Context, address.Address, *paych.SignedVoucher) (cid.Cid, error)
+
+	Send(context.Context, address.Address, types.BigInt, address.Address, types.BigInt) (cid.Cid, error)
```

Comment:
I'm not sure if this methods does enough to be here, it's basically just an alias to MpoolPushMessage which optionally uses default wallet address.

I mean `Send` is kind of important, and it's a much more obvious name than `MpoolPushMessage`, but I'd prefer renaming `MpoolPushMessage` to something else, rather than have 2 mostly identical methods

Reply:
That's a good point. I'll separate it out into another PR just for now just to get this merged in faster, but I'd also understand closing it out entirely. 

Like you mentioned, the logic for Send was just to have a go-to command for people that don't know the significance of things like gas or nonce. But they can just leave those out of the message they supply right? Not too familiar with Go.